src/main/kotlin/no/nav/helse/behandling/mvp/Sykepengegrunnlag.kt

```diff
@@ -44,6 +46,10 @@ fun vurderMVPKriterierForSykepengegrunnlaget(førsteSykdomsdag: LocalDate, perio
             .divide(BigDecimal.valueOf(3), RoundingMode.HALF_UP)
             .longValueExact(RoundingMode.HALF_UP)
 
+    if (abs((aktuellMånedsinntekt - sakskompleks.inntektsmeldinger[0].inntekt) / aktuellMånedsinntekt.toDouble()) > 0.05 ) {
```

Comment:
er dette noe vi har for vår egen del, for å gjøre det (potensielt) enklere?

Reply:
Det var Morten som sa han ville ha sjekken